scipy/optimize/__init__.py

```diff
@@ -232,8 +232,13 @@
    optimize.root-krylov
    optimize.root-dfsane
 
-Linear programming
-==================
+Linear programming / MILP
+=========================
+
+.. autosummary::
+   :toctree: generated/
```

Comment:
You don't need to repeat this, you should be able to just put `milp` right above `linprog`.

Reply:
Will do. Not sure what I was thinking.

Reply:
Fixed in next commit.